cypress/integration/auth/emailPasswordSignUpForm.js

```diff
@@ -0,0 +1,37 @@
+const FORM_URL = '/iframe.html?id=auth-stories--basic-email-password-sign-up-form';
+const BUTTON = 'button[type="submit"]';
+const SPINNER_BACKDROP = '[data-cy="app-spinner"]';
+const NAME_INPUT = 'input[name="name"]';
+const EMAIL_INPUT = 'input[name="email"]';
+const PASSWORD_INPUT = 'input[name="password"]';
+
+describe('EmailPasswordSignUpForm', () => {
+    beforeEach(() => {
```

Comment:
can we do this in the before hook instead of in order to call it only once?

Reply:
it is necessary to do this in beforeEach beause the server config is restored to its default between tests